src/functionalTest/groovy/fi/linuxbox/gradle/download/DownloadTest.groovy

```diff
@@ -12,8 +12,8 @@ import static org.gradle.testkit.runner.TaskOutcome.SUCCESS
 import static org.gradle.testkit.runner.TaskOutcome.UP_TO_DATE
 
 class DownloadTest extends Specification {
-    private static final String minimumGradleVersion = '4.3'
-    private static final Set<String> gradleVersions = ['4.10.2', minimumGradleVersion]
+    private static final String minimumGradleVersion = '5.6.1'
```

Comment:
This seems relevant: https://discuss.gradle.org/t/regularpileproperty-vs-property-file/33074/2

And there seems to be an issue about it: https://github.com/gradle/gradle/issues/10953

Reply:
Do you need this plugin to be compiled with 6.x?

Since Gradle cannot promise that a plugin compiled with certain version of Gradle will behave correctly under earlier versions of Gradle, I'm considering if this plugin should be compiled with 5.0 (which is only a year old).

Reply:
ah, ok, got it. 

So I'm changing the gradle version to the 5.0.

Most of the test are now working. I just need to fix some tests (like "Unknown host error" test) which, for an unknown reason, doesn't work yet. 

I'm trying to fix that and update the pull request before the end of the week.

Reply:
Ok, found: in the 5.0, error are in the standard output while in other version, it's in the error output 🤦‍♂️

Reply:
Updated with gradle 5.0.

Reply:
> Ok, found: in the 5.0, error are in the standard output while in other version, it's in the error output 🤦‍♂️

😅